Assets/Scripts/Augment/GunFactory.cs

```diff
@@ -53,19 +54,24 @@ public static GameObject InstantiateGun(Item bodyPrefab, Item barrelPrefab, Item
         if (displayGun.GunController.HasRecoil)
             firstPersonGunController.onFire += displayGun.GunController.PlayRecoil;
 
-        if (displayGun.GunController.projectile.GetType() == typeof(BulletController))
+        // Bypass the layer disabling here
+        // actually it probably isn't disabling, so keep it!
+        //if (isNetworkGun)
+            //return gun;
```

Comment:
Maybe remove if not in use, or are you planning on iterating on this in a future PR?

Reply:
Whoops, I forgor 💀

Reply:
Removed 🫡